views/layout/main.php

```diff
@@ -55,9 +57,9 @@
         ->options(['class' => 'navbar-nav mx-auto'])
         ->items(
             [
-                ['label' => 'Blog', 'url' => $urlGenerator->generate('blog/index')],
+                ['label' => 'Blog', 'url' => $urlGenerator->generate('blog/index'), 'active' => in_array($currentRoute, ['blog/index', 'blog/add', 'blog/archive/index', 'blog/archive/month', 'blog/archive/year', 'blog/edit', 'blog/post', 'blog/tag'])],
```

Comment:
```suggestion
                ['label' => 'Blog', 'url' => $urlGenerator->generate('blog/index'), 'active' => StringHelper::startsWith($currentRoute, 'blog/')],
```

Reply:
But don't forget to `use` the helper.

Reply:
Actually, that would also pull in `blog/comment/index`, which I excluded on purpose (it's the next menu entry)

Reply:
Well, for that `!== 'blog/comment/index'` condition could be added. Still better than keeping the list up to date.

Reply:
Fair enough. I was not aware that part exists 👍🏻

Reply:
Fair enough